server/libs/domain/src/media/media.service.ts

```diff
@@ -48,21 +48,35 @@ export class MediaService {
     try {
       const resizePath = this.storageCore.getFolderLocation(StorageFolder.THUMBNAILS, asset.ownerId);
       this.storageRepository.mkdirSync(resizePath);
-      const jpegThumbnailPath = join(resizePath, `${asset.id}.jpeg`);
+      const jpegThumbnailPath = join(resizePath, `${asset.id}.jpg`);
 
       const thumbnailDimension = 1440;
       if (asset.type == AssetType.IMAGE) {
-        try {
-          await this.mediaRepository.resize(asset.originalPath, jpegThumbnailPath, {
+        await this.mediaRepository
+          .resize(asset.originalPath, jpegThumbnailPath, {
             size: thumbnailDimension,
             format: 'jpeg',
+          })
+          .catch((error) => {
```

Comment:
This repository seems to use await rather than callbacks, is there any reason to deviate from this?

Reply:
Mostly that this is a chain of fallbacks and i didn't want nested try/catch statements.